models/pinterest_error.go

```diff
@@ -21,7 +23,9 @@ func (e *PinterestError) Error() string {
 // PinterestError if one should be returned.
 func WrapPinterestError(httpResponse *http.Response, bodyResponse *Response, err error) error {
 	if err != nil {
-		return err
+		if _, ok := err.(wrecker.ResponseError); !ok {
```

Comment:
This is a change required for the upgrade to `Wrecker v0.1.4`.  Wrecker now returns an error in the event of an HTTP error.  We need to ignore this, as we are performing the check below.